webpack/ForemanInventoryUpload/SubscriptionsPageExtension/SubscriptionsPageExtensionActions.js

```diff
@@ -7,7 +7,7 @@ export const subscriptionCountListener = (itemsCount, unsubscribe) => {
   const showToast = autoUploadEnabled => {
     !autoUploadEnabled &&
       window.tfm.toastNotifications.notify({
-        message: __('Now you can enable Red Hat inventory upload'),
+        message: __('Now you can disable Red Hat inventory upload'),
```

Comment:
@ShimShtein do you think we should remove that toast?

Reply:
Yes, I think this is much better option than the toast. The toast was a hack to work around the lack of proper extension point in katello.